qemu/tests/cfg/throttle_parameter_test.cfg

```diff
@@ -0,0 +1,108 @@
+- throttle_parameter_test:
+    type = throttle_parameter_test
+    qemu_force_use_drive_expression = no
+    images += "  stg1 stg2 stg3 stg4"
+    remove_image = yes
+    force_create_image_image1 = no
+    force_create_image = yes
+
+    image_name_stg1 = images/stg1
+    image_name_stg2 = images/stg2
+    image_name_stg3 = images/stg3
+    image_name_stg4 = images/stg4
+    image_size_stg1 = 11G
+    image_size_stg2 = 12G
+    image_size_stg3 = 13G
+    image_size_stg4 = 14G
+
+    serial_stg1="TARGET_DISK1"
```

Comment:
add space  'a = b', and the same for lin 19-21, , line 28, line 37-38, line 107-108